src/stylesheets/components/_identifier.scss

```diff
@@ -5,10 +5,19 @@ $identifier-link-gap: 1.5;
 $identifier-link-gap-desktop: 1;
 $identifier-links-gap: 4 !default;
 
+@mixin identifier-primary-link {
+  @include set-link-from-bg(
+    $theme-identifier-background-color,
+    $theme-identifier-primary-link-color,
+    $theme-color-base-darker
+  );
+}
+
 @mixin identifier-secondary-link {
   @include set-link-from-bg(
     $theme-identifier-background-color,
-    $theme-identifier-secondary-link-color
+    $theme-identifier-secondary-link-color,
+    $theme-color-base-dark
```

Comment:
Both primary and secondary link mixin colors for `set-link-from-bg()` have been updated for clarity. By default it's a dark theme and we want the links to be base-light family.

On light themes they should be the opposite, instead of the primary link color.